src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        List<Integer> supplyAmounts = new ArrayList<>();
+        List<Integer> buyAmounts = new ArrayList<>();
+
+        try (BufferedReader fromFile = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+
+            while ((line = fromFile.readLine()) != null) {
```

Comment:
Consider using constants for the operation types 'supply' and 'buy' to make the code more readable and maintainable. This way, you can easily change the value in one place if needed.